lib/utils/status.js

```diff
@@ -6,16 +6,16 @@ const runOpen = require('./runOpen');
 function status(uri, options, logger, useColor) {
   logger.info(`Project is running at ${colors.info(useColor, uri)}`);
 
-  if (options.dev && options.dev.publicPath) {
-    logger.info(
-      `webpack output is served from ${colors.info(
-        useColor,
-        options.dev.publicPath
-      )}`
-    );
-  }
-
   if (options.static && options.static.length > 0) {
+    if (typeof options.static[0].publicPath !== 'undefined') {
+      logger.info(
+        `webpack output is served from ${colors.info(
+          useColor,
+          options.static[0].publicPath
+        )}`
+      );
+    }
```

Comment:
It is different public paths :smile: I think we can output both, need very helpful message

Reply:
I see, I'll check the `output.publicPath` together.

Reply:
I think we should keep `options.dev && options.dev.publicPath` and improve static output like:
```
Content not from webpack is served from 'directory' by `publicPath` URL
```